scripts/gen_protos.py

```diff
@@ -68,13 +68,22 @@ def fix_generated_output(base_path: Path):
                 f.write(content)
     # Write init
     with (base_path / "__init__.py").open("w") as f:
+        # Add docstring to API's init
+        if str(base_path.as_posix()).endswith("/temporal/api"):
+            f.write('"""gRPC API."""\n')
```

Comment:
Why?

Reply:
Because we fail my `poe lint-docs` test because it's a package with no API. Notice right now at https://python.temporal.io/ we don't have a summary doc for just the `temporal.api` package. That's because I had traditionally gen'd docs after protos and didn't have this here.